admin/customers.php

```diff
@@ -1164,7 +1164,7 @@ function check_form() {
 
       if ((!isset($_GET['cID']) || (isset($_GET['cID']) && ($_GET['cID'] == $customer['customers_id']))) && !isset($cInfo)) {
         $country = $db->Execute("select countries_name
-                                 from " . TABLE_COUNTRIES . "
+                                 from " . TABLE_COUNTRIES_NAME . "
                                  where countries_id = '" . (int)$customer['entry_country_id'] . "'");
 
```

Comment:
Should this include the Language ID? ... else line 1173 is only pulling the first randomly-selected country-name record found.